sources/packages/backend/libs/sims-db/src/entities/assessment-trigger.type.ts

```diff
@@ -51,4 +51,9 @@ export enum AssessmentTriggerType {
    * to be adjusted.
    */
   RelatedApplicationChanged = "Related application changed",
+  /**
+   * Triggered manually by a ministry user. The application must not be archived and the
+   * current assessment status must be completed.
```

Comment:
I am not sure if the conditions such as application must not be archived is required for enum description.